### DISCUSSION TITLE
Missing guidance on GenAIOps (LLMOps) with Semantic Kernel / especially in the area of ​​.NET + experimentation and evaluation

### DISCUSSION THREAD

**Discussion by RicardoNiepel** [microsoft/semantic-kernel#10022] [Q&A] [answered]
This discussion adds to #10007 with additional dimensions.

Looking into [Microsoft's current strategy for GenAIOps (LLMOps)](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/concept-llmops-maturity) Prompt Flow as part of AI Studio and Azure Machine Learning is always used as a solution ([1](https://github.com/microsoft/genaiops-promptflow-template) & [2](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/how-to-end-to-end-llmops-with-prompt-flow))

But concrete **guidance is missing**:
- How to do **GenAIOps** with **Semantic Kernel Agent Framework**
  - especially how to do **prompt experimentation and evaluation** when using **Agent Collaboration with AgentChat** is not available
- How to do **GenAIOps** when using **DotNet with Semantic Kernel**, as Promptflow lags .NET support [#1317 ](https://github.com/microsoft/promptflow/issues/1317#issuecomment-2151313898)

_Any PG insights, community input or references are welcome._

**Comment by sophialagerkranspandey** [comment, marked as answer]
Created an issue to triage here: https://github.com/microsoft/semantic-kernel/issues/10892
